staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/generated.proto

```diff
@@ -327,8 +327,10 @@ message CustomResourceSubresourceScale {
 
   // LabelSelectorPath defines the JSON path inside of a CustomResource that corresponds to Scale.Status.Selector.
   // Only JSON paths without the array notation are allowed.
-  // Must be a JSON Path under .status.
   // Must be set to work with HPA.
+  // The field pointed by this JSON path must be a string field (not a complex selector struct) and contain
```

Comment:
Need we add a comment: Must be a JSON Path under .spec or .status?

Reply:
added. Thanks